packages/core/src/routes/interaction/single-sign-on.ts

```diff
@@ -147,13 +146,6 @@ export default function singleSignOnRoutes<T extends IRouterParamContext>(
         new RequestError({ code: 'auth.forbidden', status: 403 })
       );
 
-      const registerEventUpdateLog = createLog(`Interaction.Register.Update`);
-      registerEventUpdateLog.append({ event: 'register' });
```

Comment:
Why do we remove logging from this API?

Reply:
This is the interaction event update log. 
Same as the following logic, should be triggered by calling `PUT /interaction/event`.